### PR TITLE
Chore/generate changelog

### PR DESCRIPTION
**Pull request by jimmyfortinx**
Add commitlint, conventional-changelog and commitizen to the project to standardize commit messages and generate a changelog.

Remove grunt-bump to let the `npm version` do all the work to update the bower file version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -113,6 +116,9 @@
     "karma-script-launcher": "~0.1.0",
     "less-plugin-remcalc": "^0.0.1",
     "lodash": "^3.0.1",
+    "npm-bower-sync-ver": "^1.1.2",
+    "npm-scripts-config": "^0.0.2",
+    "npm-scripts-conventional-changelog": "^0.1.0",
```
Comment (Jisay) [inline package.json]: I noticed here that you're using your own npm packages. Will they be migrated to the GitHub ovh-ux organization?

Comment (jimmyfortinx) [reply package.json]: Yes they should be migrated to ovh-ux organization.

**Comment by jimmyfortinx** [comment]
@Jisay yarn.lock is pushed on all other managers and soon it will be used on the build process by using worker cache, so I see the need to push it on the repo.

**Comment by antleblanc** [comment]
@jimmyfortinx according to the `Makefile` which only using `npm` commands for the moment, we should generate it on the right time.

**Pull request merged**